renovate.json

```diff
@@ -19,8 +19,8 @@
   "labels": [
     "release-note-none"
   ],
-  "extends": [
-    ":gitSignOff"
+  "ignorePaths": [
+    "modules/tests/**"
```

Comment:
I don't see the `modules/tests` directory. Is it generated and not comitted to the repo?

Reply:
This path modules/tests/ exists only in releases 4.15 and older. The 4.16 is using different project structure (and this rule is not affecting anything).

Reply:
ok